addons/ui/fnc_progressBar.sqf

```diff
@@ -89,4 +80,70 @@ if (!isNil QGVAR(ProgressBarParams)) then {
 
 GVAR(ProgressBarParams) = [_arguments, _condition, _onSuccess, _onFailure, CBA_missionTime, _totalTime, _blockMouse, _blockKeys, _allowClose];
```

Comment:
Not that it really matters, but you could set these on the display itself and that way it get's cleaned up automatically

Reply:
The display dies when saving and loading a savegame.
If I use a global instead, which is serialized properly, I can create a new loading bar with the same parameters once the game is loaded:
https://github.com/CBATeam/CBA_A3/pull/912/files#diff-8ff25e6d6abfb5db687c39f11a07e5b4R7

Only thing that doesn't work is the block input display, but I just assume that this only applies to mouse enabled progress bars, since